pytools/graph.py

```diff
@@ -178,24 +178,31 @@ def __init__(self, node):
         self.node = node
 
 
-def compute_topological_order(graph):
+def compute_topological_order(graph, key=None):
     """Compute a toplogical order of nodes in a directed graph.
 
     :arg graph: A :class:`collections.abc.Mapping` representing a directed
         graph. The dictionary contains one key representing each node in the
         graph, and this key maps to a :class:`collections.abc.Iterable` of
         nodes that are connected to the node by outgoing edges.
 
+    :arg key: A custom key function may be supplied to determine the order in
```

Comment:
Should document the calling signature of the function.

Reply:
Done.